src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -46,6 +48,9 @@
     protected List<FeedbackResponseAttributes> responsesToSave = new ArrayList<>();
     protected List<FeedbackResponseAttributes> responsesToDelete = new ArrayList<>();
     protected List<FeedbackResponseAttributes> responsesToUpdate = new ArrayList<>();
+    protected List<FeedbackResponseCommentAttributes> commentsToSave = new ArrayList<>();
+    protected List<FeedbackResponseCommentAttributes> commentsToUpdate = new ArrayList<>();
+    protected Map<String, String> commentIndxToResponseReceiverMap = new HashMap<>();
```

Comment:
There is no need to have a map here.
You can get the recipient by using
https://github.com/TEAMMATES/teammates/blob/444c0920576e42dbc3d6d6588fc696b500a26d9a/src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java#L330-L333